playwright/async_api.py

```diff
@@ -4913,7 +4913,9 @@ async def clearPermissions(self) -> NoneType:
         """
         return mapping.from_maybe_impl(await self._impl_obj.clearPermissions())
 
-    async def setGeolocation(self, geolocation: typing.Dict) -> NoneType:
+    async def setGeolocation(
+        self, latitude: int, longitude: int, accuracy: int = None
+    ) -> NoneType:
         """BrowserContext.setGeolocation
 
         Sets the context's geolocation. Passing `null` or `undefined` emulates position unavailable.
```

Comment:
`Passing null or undefined emulates position unavailable` - how is this working now?

Reply:
Probably I have to revert the changes so that its possible again to specify None for the geolocation argument.

Reply:
@dgozman was saying it did not necessarily work due to caching in the browsers, but I'd say staying consistent with upstream is a good idea for now.